### ISSUE TITLE
Right Click to move to Specific List

### ISSUE THREAD

**Issue by rismay** [imdone/imdone#7]
**Is your feature request related to a problem? Please describe.**
I have a huge "DONE" list. I hide that list and created a smart "Recently Done" list. However, this makes it impossible for me to drag and drop into the "Recently Done" list. 

**Describe the solution you'd like**
Right click a card and give it the ability to move to any list I would like.

**Comment by piascikj** [comment]
@rismay Have you tried the "ignore" option for your DONE list?  This would make any card you drag into your DONE list disappear from the board.
![Kapture 2020-02-06 at 21 06 06](https://user-images.githubusercontent.com/233505/74000268-b0ad0200-4924-11ea-9e9b-5a4a20b57006.gif)

**Comment by rismay** [comment]
@piascikj Yes I have. In that case I still have two columns: A "Recently Done" smartlist and a "Done" list showing no items. I would like to not have two columns showing and not to have to go into the card to edit the list.

**Comment by trevormeier** [comment]
It would also be nice to be able to specify which board a card belongs to while editing inside the app. It's easy enough when editing files externally but there's no way to do it without click & drag inside IMDONE at the moment
